Applications/DisplayProperties/main.cpp

```diff
@@ -12,12 +16,28 @@ int main(int argc, char** argv)
 
     auto window = GWindow::construct();
     window->set_title("Display Properties");
-    window->move_to(100,100);
+    window->move_to(100, 100);
     window->resize(400, 448);
     window->set_resizable(false);
     window->set_main_widget(instance.root_widget());
     window->set_icon(load_png("/res/icons/16x16/app-display-properties.png"));
 
+    // Let's create the menubar first
+    auto menubar = make<GMenuBar>();
+
+    auto app_menu = GMenu::construct("Display Properties");
+    app_menu->add_action(GAction::create("Exit", { Mod_Alt, Key_F4 }, [&](const GAction&) {
+        app.quit();
+    }));
+    menubar->add_menu(move(app_menu));
+
+    auto help_menu = GMenu::construct("Help");
+    help_menu->add_action(GAction::create("About", [&](const GAction&) {
+        GAboutDialog::show("Terminal", load_png("/res/icons/32x32/app-display-properties.png"), window);
```

Comment:
"Terminal"? ;)

Reply:
Whoops, I was using `Terminal`'s `main.cpp` for inspiration. Looks like I verbatim copied it.